js-core-2/lesson7.md

```diff
@@ -98,6 +99,48 @@ printArray(student);
 // "name is Mohamed, age is 22, interests are ["javascript", "c#"]"
 ```
 
+## More advanced working with objects
```

Comment:
for some reason GitHub is not formatting this for me properly 

![image](https://user-images.githubusercontent.com/5975076/28118674-dfa925ca-6709-11e7-99d6-942eae81b62b.png)